File: codewars/simple-pig-latin/kata.js
function pigIt(str){
  const sep = " ";
  const suffix = "ay";
  return str.split(sep).map(word => {
    if (!/[a-z]/i.test(word)) return word;
    return word.slice(1) + word[0] + suffix;
  }).join(sep);
}

File: codewars/simple-pig-latin/kata.rb
def pig_it text
  sep = " "
  text.split(sep).map { |word| pig_it_word word }.join(sep)
end

def pig_it_word word
  return word unless /\w/i =~ word
  suffix = "ay"
  word[1..-1] + word[0] + suffix;
end

p pig_it('Pig latin is cool')
